src/connect.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo -e 'connect 5C:BA:37:26:6D:9A\nquit' | bluetoothctl
```

Comment:
bluetoothctl also requires superuser. It's probably not good practice to require the user to remember the sudo every time, especially when the error you get for not having sudo is  "No default controller available." That's confusing and I wouldn't think "oh, I just forgot sudo." Maybe make it "sudo bluetoothctl"